pyproject.toml

```diff
@@ -39,7 +39,7 @@ dependencies = ["ase", "joblib", "phonopy", "pymatgen"]
 version = "0.0.4"
 
 [project.optional-dependencies]
-models = ["chgnet", "mace-torch", "matgl>=0.9.0"]
+models = ["chgnet", "mace-torch", "matgl>=0.9.0", "sevenn"]
```

Comment:
let's add a minimum version pin to all models pointing at the latest pypi version in all cases. older versions have known bugs in many cases and should not be used

Reply:
Will do

Reply:
Pinned